Flick/Controllers/DiscoverViewController.swift

```diff
@@ -31,6 +44,29 @@ class DiscoverViewController: UIViewController {
             make.leading.trailing.equalToSuperview().inset(24)
             make.height.equalTo(36)
         }
+        
+        discoverFeedTableView.snp.makeConstraints { make in
+            make.top.equalTo(searchBar.snp.bottom).offset(16)
+            make.leading.trailing.bottom.equalToSuperview()
+        }
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
```

Comment:
i think it might be better to do `viewDidAppear` so there's no wait in showing this view controller. cuz i think now it'll wait till the request is over to show this VC right